package.json

```diff
@@ -17,7 +17,7 @@
     "preview": "node ./scripts/concurrently.cjs serve:jupyterhub preview",
     "serve": "node ./scripts/concurrently.cjs serve:jupyterhub serve:vue",
     "serve:vue": "vite --mode offline",
-    "serve:jupyterhub": "nodemon -e js,mjs,cjs,json src/services/mock/json-server.cjs",
+    "serve:jupyterhub": "nodemon src/services/mock/json-server.cjs --watch src/services/mock/",
```

Comment:
Whats this? is it related to the PR?

Reply:
Not related to the PR but worth getting in IMO.

The mock JSON server was unnecessarily restarting every time any JS file was edited, when it only needs to restart when files in `src/services/mock` are changed. See https://github.com/cylc/cylc-ui/pull/1786/commits/447657b1b22736945ab9879a0beaa6936b8398f3